### ISSUE TITLE
Search box UI Update

### ISSUE THREAD

**Issue by YolineB** [data-umbrella/event-board-web#366]
Mobile view of search box should reflect the following, as seen on Figma

- [ ] Start date / End date need to be halved on mobile
- [ ] region & price split on mobile filter

[Updated 2/10 - Yoline]

**Comment by loki1205** [comment]
Hi, I would love to work with you on this one. Can you please assign this to me?

**Comment by reshamas** [comment]
> Hi, I would love to work with you on this one. Can you please assign this to me?

@loki1205 Yes, this issue is available. If you could submit a PR within the next couple of days, that would be great.

**Comment by loki1205** [comment]
Hi, I am unable to see the Figma file. I have requested access. Kindly accept it and let me know.

**Comment by reshamas** [comment]
@iliadeleon 
Are you able to add the screenshot here on GitHub?

**Comment by loki1205** [comment]
![Screenshot_20230210-050136](https://user-images.githubusercontent.com/87192195/217962434-a9af24b6-2431-4635-b120-0fabbad8e2ce.png)

**Comment by YolineB** [comment]
@loki1205 I do not have the latest version, but the search box will look similar to the screenshot below. Please note that the categories have changed to the following:

-Start Date
-End Date
-Event Type
-Topic
-Language
-Region
-Price 

Start Date/End Date and Region/Price (instead of Category / Price ) will be the two split boxes

**Comment by YolineB** [comment]
Hello @loki1205, here is an updated figma design for the search box - let me know if you have any questions! Please note that we do request PRs for good first issues to be completed by February 17.


![Image](https://user-images.githubusercontent.com/92892101/218675725-10f04b8e-5d27-44d6-8e10-34db965a769a.png)
